api/openapi.gen.yaml

```diff
@@ -258,11 +258,14 @@ components:
         cores:
           format: int32
           type: integer
-        memory:
+        memory_mib:
           format: int64
           type: integer
         name:
           type: string
+        storage_gb:
+          format: int64
+          type: integer
         supported:
           type: boolean
         vcpus:
```

Comment:
@RHEnVision/qe we are renaming memory to memory_mib to indicate that memory size is actually MiB not MB. The same for the new field storage (which is actually in GB not GiB).